packages/expect-utils/src/__tests__/utils.test.ts

```diff
@@ -670,37 +670,55 @@ describe('arrayBufferEquality', () => {
   test('returns false when given non-matching buffers', () => {
     const a = Uint8Array.from([2, 4]).buffer;
     const b = Uint16Array.from([1, 7]).buffer;
-    expect(arrayBufferEquality(a, b)).not.toBeTruthy();
+    expect(arrayBufferEquality(a, b)).toBe(false);
```

Comment:
this was a separate bug - it returned `undefined`, not `false` since `Uint16Array.from([1, 7]).buffer instanceOf ArrayBuffer` for some reason is `false.